conda-store-server/conda_store_server/action/generate_lockfile.py

```diff
@@ -13,19 +15,39 @@ def action_solve_lockfile(
     conda_command: str,
     specification: schema.CondaSpecification,
     platforms: typing.List[str] = [conda_utils.conda_platform()],
+    # https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-channels.html
+    conda_flags: str = "--strict-channel-priority",
```

Comment:
From the docs linked above:

> We recommend setting channel priority to "strict" when possible.

> channel_priority (ChannelPriority)
Accepts values of 'strict', 'flexible', and 'disabled'. The default value is 'flexible'.